dags/openshift_nightlies/tasks/install/openshift.py

```diff
@@ -69,7 +70,10 @@ def _get_task(self, operation="install", trigger_rule="all_success"):
         raise NotImplementedError()
 
     def get_install_task(self):
-        return self._get_task(operation="install")
+        indexer = StatusIndexer(self.dag, self.dag_config, self.release, "install").get_index_task()
+        install_task = self._get_task(operation="install")
+        install_task >> indexer
```

Comment:
What's the point of indexing the install task?

Reply:
it is just another metric - install duration. I do recall we had something like this in scale-ci-deploy so maybe it is redundant.

Reply:
ok, let's leave it then, indexing it would be installer agnostic and can be useful for self-managed clusters too